File: JavaSrc/toolkit/bean/CollectionValue.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="##default")
public class CollectionValue extends BaseApiBean {
  private String label;
  private String value;

  public String getLabel() {
     return this.label;
  }

  public void setLabel(String label) {
     this.label = label;
  }

  public String getValue() {
     return this.value;
  }

  public void setValue(String value) {
     this.value = value;
  }


}

File: CiscoPCCE.Toolkit/Sdk/Bean/CollectionValue.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlType("##default")]
    public class CollectionValue : BaseApiBean
    {
        [XmlElement("label")]
        public string? Label { get; set; }

        [XmlElement("value")]
        public string? Value { get; set; }
    }
}